projects/igniteui-angular/migrations/update-15_0_0/index.ts

```diff
@@ -47,4 +47,38 @@ export default (): Rule => async (host: Tree, context: SchematicContext) => {
             }
         });
     }
+
+    const updateTypographyAndScrollbar = (content: string) => {
+        const typography = /igx-typography/g,
+            scrollbar = /igx-scrollbar/g;
+        return content
+            .replace(typography, 'ig-typography')
+            .replace(scrollbar, 'ig-scrollbar');
+    };
+
+    const indexPath = '/src/index.html';
+    if (host.exists(indexPath)) {
+        host.overwrite(indexPath, updateTypographyAndScrollbar(host.read(indexPath).toString()));
+    }
+
+    update.templateFiles.forEach(path =>
+        host.overwrite(path, updateTypographyAndScrollbar(host.read(path).toString()))
+    );
+
+    const graysVar = /\$grays:\s{0,}(#\w+)(\r\n|\r|\n|,)/s,
```

Comment:
this will fail if `$grays` is set to anything but hexadecimal syntax. All these will not be replaced:
$grays: red,
$grays: rgb(204, 102, 153),
$grays: rgba(107, 113, 127, 0.8),
$grays: hsl(228, 7%, 86%),
$grays: hsla(20, 20%, 85%, 0.7),